tests/cypress/support/commands.ts

```diff
@@ -557,6 +557,9 @@ Cypress.Commands.add('removeClusterLabels', (clusterName, key, value) => {
   cy.wait(500);
   cy.clickButton('Save');
   cy.contains('Save').should('not.exist');
+  // After label removal from cluster, it says 409 (conflict error) while saving.
```

Comment:
Has this been reported as a bug?

Reply:
I would say, there is no as such bug for this behavior.

Reply:
As you found out it seems there was and I commented there: `https://github.com/rancher/dashboard/issues/9563#issuecomment-2404603657`

If this workaround helps I am ok, but pls add a comment to remove it once the issue is fixed.

Reply:
Also, tests failing non-related to this, I guess is a hiccup? Could you pls ensure other tests are not affected (77,20,30,31 etc?). Thanks.

Reply:
The tests which are failing 30,31 are related to this test failure. But the 77,20 are not related could be a hiccup.